docs/developing/integrate/wallets/cli.md

```diff
@@ -0,0 +1,101 @@
+---
+title: CLI
+proofedDate: 20241909
+iterationBy: na
+includedInSite: false
+approvedBy: na
+comment:
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+## Introduction
+
+Get the Safe CLI
+Prerequisite: Install Docker Desktop (https://www.docker.com/products/docker-desktop/)
+
+Once Docker is installed on your system, create Safe account using the Safe guide
+
+## How to create a new Safe CLI
+
+### Step 1: Create 2 or more wallets
+
+1.1 Go to the [login](https://safe.neonevm.org/welcome) page and connect your wallet and create two or more wallets
+
+<div className='neon-img-width-600' style={{textAlign: 'center'}}>
+
+![](img/cli1.png)
+
+</div>
+
+### Step 2: Installation
+2.1 Run
+
+docker run -it safeglobal/safe-cli safe-creator
+
```

Comment:
Remove this "2.1" point because we can't create Safe account with CLI. It throws error.

Reply:
ready, new version is added